### DISCUSSION TITLE
Serverless Solution for deploying Agent with Persistent WebSocket Support

### DISCUSSION THREAD

**Discussion by younessssssss** [ajndkr/lanarky#125] [Q&A]
Hello,

I'm building a langchain agent and need to stream the agent execution steps using Websockets. Can anyone recommend a suitable Serverless platform or share best practices for this setup?

I tried Modal but it does not work the connection needs to be persistent and Modal does not support persistant container 

Thanks!

**Comment by pietz** [comment]
Other might have better ideas but Azure Functions does support a FastAPI backend, so this should work. What will not work at the moment is streaming, because they don't support it.

**Reply by younessssssss** [reply]
Thank you ,i will take a look at it.
And do not need streaming just persistent communication.
